samples/python/bulk_edit_issues.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+
+page_size = 1000 #max value
+page = 0
+api_key = "<api-key>"
+endpoint = "https://api.helpshift.com/v1/<domain>/issues"
+
+def bulk_edit_issue(issues):
+    print ("Triggering Bulk Action for " + str(len(issues)) + " issues")
```

Comment:
This can also contain information about the issue-ids on which bulk action is being applied.

Reply:
For a lot of issues, the print elements looked really chaotic to be enabled by default.
IMO if someone wants to check for issue ids explicitly, they can add a log line that shows them that.

Reply:
IMO, it is should print at least some information that can be helpful in case some error occurs while executing this script. Even the start and end `issue-id` can be useful since the issues are sorted.

Reply:
The raise error part is to pinpoint a server error. 
Here the limit is 1000 and I am adding another piece of code where it takes 5000 values in one instance - which is more chaotic. 
But I can add the start and end of the issue.

Reply:
The aim behind adding information is that the user should be able to figure out where to start the script again in case some error occurred in the previous run. Giving at least, start and end `issue-ids` narrow it down.